### PR TITLE
py-pyvista: fix import tests

### PR DESCRIPTION
**Pull request by adamjstewart**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-pyvista/package.py [hunk 1]
```diff
@@ -12,6 +12,9 @@ class PyPyvista(PythonPackage):
     homepage = "https://github.com/pyvista/pyvista"
     pypi = "pyvista/pyvista-0.32.1.tar.gz"
 
+    # Requires optional trame dependency
+    skip_modules = ["pyvista.trame"]
+
```
Comment (banesullivan) [inline var/spack/repos/builtin/packages/py-pyvista/package.py]: I'd maybe also skip `pyvista.ext`

Comment (banesullivan) [reply var/spack/repos/builtin/packages/py-pyvista/package.py]: And maybe also `pyvista.jupyter`, though this is not needed now. Future versions of PyVista may do things in that module that require jupyter or trame dependencies.

**Review by banesullivan — APPROVED** [review]
LGTM

**Review by tldahlgren — APPROVED** [review]
I am unable to independently test this due to problems building `libtheora` so I will go with what you two consider correct.

**Pull request merged**